File: app/src/main/java/com/doordash/binding/viewmodel/DishModel.java
package com.doordash.binding.viewmodel;

import androidx.databinding.BaseObservable;

import com.doordash.model.data.PopularItem;

public class DishModel extends BaseObservable {

    PopularItem item;

    public DishModel(PopularItem item) {
        this.item = item;
    }

    public String getDishImageUrl(){
        return item.getImgUrl();
    }
    public String getDishName(){
        return item.getName();
    }

    public String getDishDesciption(){
        return item.getDescription();
    }
}

File: app/src/main/java/com/doordash/binding/viewmodel/DishModel.kt
package com.doordash.binding.viewmodel

import androidx.databinding.BaseObservable
import com.doordash.bean.PopularItem

class DishModel(var item: PopularItem) : BaseObservable() {
    val dishImageUrl: String
        get() = item.imgUrl

    val dishName: String
        get() = item.name

    val dishDesciption: String
        get() = item.description

}